tensorflow/stream_executor/rocm/BUILD

```diff
@@ -140,126 +141,137 @@ cc_library(
+        "//tensorflow/stream_executor:scratch_allocator",
+        "//tensorflow/stream_executor/gpu:gpu_activation",
+        "//tensorflow/stream_executor/gpu:gpu_helpers_header",
+        "//tensorflow/stream_executor/gpu:gpu_executor_header",
+        "//tensorflow/stream_executor/gpu:gpu_stream_header",
+        "//tensorflow/stream_executor/gpu:gpu_kernel_header",
+        "//tensorflow/stream_executor/lib",
+        "//tensorflow/stream_executor/platform",
+        "//tensorflow/stream_executor/platform:dso_loader",
+        "@local_config_rocm//rocm:rocm_headers",
+    ] + if_static([
+        "@local_config_rocm//rocm:rocfft",
+    ])),
+    alwayslink = True,
+)
 
-# FIXME: enable in future PRs
-#cc_library(
-#    name = "miopen_plugin",
-#    srcs = ["rocm_dnn.cc"],
-#    hdrs = [],
-#    copts = [
-#        # STREAM_EXECUTOR_CUDNN_WRAP would fail on Clang with the default
-#        # setting of template depth 256
-#        "-ftemplate-depth-512",
-#    ],
-#    visibility = ["//visibility:public"],
-#    deps = [
-#        ":rocm_diagnostics",
-#        ":rocm_driver",
-#        ":rocm_gpu_executor",
-#        ":rocm_platform_id",
-#        "//third_party/eigen3",
-#        "//tensorflow/core:lib",
-#        "//tensorflow/core:lib_internal",
-#        "//tensorflow/core:logger",
-#        "//tensorflow/stream_executor:dnn",
-#        "//tensorflow/stream_executor:event",
-#        "//tensorflow/stream_executor:logging_proto_cc",
-#        "//tensorflow/stream_executor:plugin_registry",
-#        "//tensorflow/stream_executor:scratch_allocator",
-#        "//tensorflow/stream_executor:stream_executor_pimpl_header",
-#        "//tensorflow/stream_executor:temporary_device_memory",
-#        "//tenosrflow/stream_executor/gpu:gpu_activation_header",
-#        "//tenosrflow/stream_executor/gpu:gpu_stream_header",
-#        "//tenosrflow/stream_executor/gpu:gpu_timer_header",
-#        "//tensorflow/stream_executor/lib",
-#        "//tensorflow/stream_executor/platform",
-#        "//tensorflow/stream_executor/platform:dso_loader",
-#        "@com_google_absl//absl/strings",
-#        "@local_config_rocm//rocm:rocm_headers",
-#    ] + tf_additional_miopen_plugin_deps() + if_static(["@local_config_rocm//rocm:miopen"]),
-#    alwayslink = True,
-#)
+cc_library(
+    name = "miopen_plugin",
+    srcs = if_rocm_is_configured(["rocm_dnn.cc"]),
+    hdrs = if_rocm_is_configured(["rocm_dnn.h"]),
+    copts = [
+        # STREAM_EXECUTOR_CUDNN_WRAP would fail on Clang with the default
+        # setting of template depth 256
+        "-ftemplate-depth-512",
+    ],
+    visibility = ["//visibility:public"],
+    deps = if_rocm_is_configured([
+        ":rocm_diagnostics",
+        ":rocm_driver",
+        ":rocm_gpu_executor",
+        ":rocm_platform_id",
+        "//third_party/eigen3",
+        "//tensorflow/core:lib",
+        "//tensorflow/core:lib_internal",
+        "//tensorflow/core:logger",
```

Comment:
@deven-amd per test results it's better to remove this dependency.

Reply:
done